test/lit/fuzz-types/nominal.test

```diff
@@ -20,3 +20,27 @@
 ;; CHECK-NEXT: (type $17 (array (mut (ref $2))))
 ;; CHECK-NEXT: (type $18 (array (ref $10)))
 ;; CHECK-NEXT: (type $19 (struct_subtype (field (mut i16) (ref i31) f32 f32 f64 (mut f32)) $0))
+;; CHECK-NEXT:
+;; CHECK-NEXT: Inhabitable types:
+;; CHECK-NEXT:
+;; CHECK-NEXT: Built 20 types:
+;; CHECK-NEXT: (type $0 (type $0 (struct (field (mut i16) i31ref f32 f32 f64))))
+;; CHECK-NEXT: (type $1 (type $1 (func (param f64 v128))))
+;; CHECK-NEXT: (type $2 (type $2 (struct (field (mut (ref null $19)) f64 arrayref))))
+;; CHECK-NEXT: (type $3 (type $3 (struct_subtype (field (mut i16) (ref i31) f32 f32 f64) $0)))
+;; CHECK-NEXT: (type $4 (type $4 (struct)))
+;; CHECK-NEXT: (type $5 none)
+;; CHECK-NEXT: (type $6 (type $6 (array (mut eqref))))
+;; CHECK-NEXT: (type $7 (type $7 (func_subtype (param f64 v128) $1)))
+;; CHECK-NEXT: (type $8 (type $8 (array anyref)))
+;; CHECK-NEXT: (type $9 (type $9 (array f32)))
+;; CHECK-NEXT: (type $10 (type $10 (struct_subtype (field (mut i16) (ref i31) f32 f32 f64) $0)))
+;; CHECK-NEXT: (type $11 (type $11 (func (result f64))))
+;; CHECK-NEXT: (type $12 (type $12 (func_subtype (param f64 v128) $1)))
+;; CHECK-NEXT: (type $13 (type $13 (func_subtype (param f64 v128) $12)))
+;; CHECK-NEXT: (type $14 (type $14 (func_subtype (result f64) $11)))
+;; CHECK-NEXT: (type $15 (type $15 (func_subtype (result f64) $14)))
+;; CHECK-NEXT: (type $16 (type $16 (func (param (ref struct)) (result structref))))
+;; CHECK-NEXT: (type $17 (type $17 (array (mut (ref $2)))))
+;; CHECK-NEXT: (type $18 (type $18 (array (ref $10))))
+;; CHECK-NEXT: (type $19 (type $19 (struct_subtype (field (mut i16) (ref i31) f32 f32 f64 (mut f32)) $0)))
```

Comment:
Other than that, these results look identical to before - was any work done to make them inhabitable?

Reply:
Unfortunately these seeds don't require much fixing up. I commented separately to point out the one change I did notice.

The fuzzing was extremely effective in catching bugs with this implementation, though. Most bugs were surfaced with just a handful of iterations. Now the fuzzer runs for hundreds of thousands of iterations without finding new problems.